test/gpu_reference_kernel.cpp

```diff
@@ -980,9 +1057,16 @@ struct gpu_reference_conv_3d : gpu_reference_kernel_base
                 EXPECT(hipMemcpy(
                            in_dev, in.data.data(), sizeof(TRef) * in_sz, hipMemcpyHostToDevice) ==
                        hipSuccess);
+                // TODO(Amber): copy output before computation because output may
```

Comment:
ref

Reply:
[resolved]

@amberhassaan Please hide all [resolved] comments (I can't do that), thanks.